### PR TITLE
Support netconf:base:1.1

### PR DESCRIPTION
**Pull request by jonnary**
This patch support netconf:base:1.1 version.
Also we will compat netconf:base:1.0 version.

### CODE REVIEW

**Comment by jonnary** [comment]
Any body can help review the commit:)?

**Comment by jonnary** [comment]
Any body can help review the commit:)?

**Comment by pplavetzki** [comment]
@jonnary I can help.  Can you pass me some information on how I can test this?

**Comment by jonnary** [comment]
Thanks. You can check the push with the switch which support netconf:base:1.1 version.

**Comment by vnitinv** [comment]
@mgireeshk can you please review the code.

File: netconf/transport.go [hunk 1]
```diff
@@ -39,27 +41,52 @@ type Transport interface {
...
+	dataInfo = append(dataInfo, seperator...)
+	t.Write(dataInfo)
+
 	return nil // TODO: Implement error handling!
 }
 
 func (t *transportBasicIO) Receive() ([]byte, error) {
-	return t.WaitForBytes([]byte(msgSeperator))
+	var seperator []byte
+	if t.version == "v1.1" {
```
Comment (mgireeshk) [inline netconf/transport.go:84]: Good to have version as enum rather than string

**Review by mgireeshk — APPROVED** [review]

File: netconf/transport.go (hunk 1 above)
Comment (vnitinv) [reply netconf/transport.go:84]: we will take care of this in the next code change. For now, we should merge this.

**Pull request merged**